package.json

```diff
@@ -191,6 +191,7 @@
     "store": "2.0.12",
     "striptags": "2.2.1",
     "superagent": "3.8.3",
+    "swiper": "4.4.2",
```

Comment:
Latest `swiper` would be `4.4.6` that seems to fix some bugs:

https://github.com/nolimits4web/swiper/blob/c6f92692d2643d41cf14638638b991b5cd41d1e2/CHANGELOG.md#readme